user-guide.md

```diff
@@ -86,6 +106,45 @@ Make sure you installed JupyterLab Desktop following the steps outlined above in
   %pip install scikit-learn
   ```
 - In order to use the newly installed package you need to restart your active notebook's kernel or create a new notebook
+- If you install a new JupyterLab extension with UI components, then you will need to create a new session for changes to take effect
```

Comment:
I wonder if a restart button would be useful for this use-case. What I've been doing is opening the environment selection list and clicking enter to re-load server with the current environment, but with a dedicated button it could save me three click (and be obvious to every user).

Reply:
maybe we can extend environment select popup into server info popup, it can have restart button. also features like copy server URL to clipboard. it could also just go into the app menu.